modules/scheduler/batch-job-template/main.tf

```diff
@@ -66,7 +66,7 @@ locals {
 
 module "instance_template" {
   source  = "terraform-google-modules/vm/google//modules/instance_template"
-  version = "> 7.6.0"
+  version = "~> 8.0"
```

Comment:
[This](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#terraform-core-and-provider-versions) states to only constrain on a minimum. Thoughts?

I agree it seems kind of crazy to silently allow upgrade on major version without some intervention.

Reply:
Here is the text I read

- When depending on third-party modules, require specific versions to ensure that updates only happen when convenient to you.
- For modules maintained within your organization, specifying version ranges may be appropriate if semantic versioning is used consistently or if there is a well-defined release process that avoids unwanted updates.

I believe bullet 2 is what applies to us.